dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/SdkFeature.kt

```diff
@@ -96,14 +96,16 @@ internal class SdkFeature(
         wrappedFeature.onInitialize(context)
 
         if (wrappedFeature is StorageBackedFeature && dataUploadConfiguration != null) {
-            setupUploader(wrappedFeature.requestFactory, dataUploadConfiguration)
+            setupUploader(wrappedFeature, dataUploadConfiguration)
         }
 
         if (wrappedFeature is TrackingConsentProviderCallback) {
             coreFeature.trackingConsentProvider.registerCallback(wrappedFeature)
         }
 
         initialized.set(true)
+
+        uploadScheduler.startScheduling()
```

Comment:
this function name is misleading here now, maybe we should have a different interface here, this is not scheduling anything here now.

Reply:
I would keep it for consistency with `stopScheduling` and class name - `UploadScheduler`